pyproject.toml

```diff
@@ -71,21 +72,20 @@ docs = [
     "sphinx-togglebutton>=0.3.2",
     #    "litestar-sphinx-theme @ {root:uri}/../litestar-sphinx-theme", # only needed when working on the theme
     "litestar-sphinx-theme @ git+https://github.com/litestar-org/litestar-sphinx-theme.git@v3",
-]
-lint = [
+    # - Linting
     "codecov-cli",
     "mypy == 1.11.2",
     "pre-commit",
     "pyright",
     "ruff",
     "slotscheck",
-]
-test = [
+    # - Testing
     "covdefaults",
     "pytest",
     "pytest-cov",
 ]
 
+# TODO: needs migrating. Poe?
```

Comment:
do these need to exist? each basically has a corresponding `make` command

Reply:
it can go away for now. i think the idea was to shuffle things into pyproject and lighten the Make targets

Reply:
+1 for `Makefile` entry.